src/popup-page/modules/app.tsx

```diff
@@ -0,0 +1,57 @@
+import React, { useEffect, useContext } from 'react'
+import { getItem } from '../../utils/storage'
+import { GlobalContext } from '../context/global'
+
+function BeeApiUrlChangeForm(): JSX.Element {
+  const globalStateContext = useContext(GlobalContext)
+  const { dispatch: changeGlobalState, state: globalState } = globalStateContext
+
+  const handleSubmit = (event: React.FormEvent<HTMLElement>): void => {
+    event.preventDefault()
+
+    if (!/^https?:\/\/.*/i.test(globalState.beeApiUrl)) {
+      // eslint-disable-next-line no-alert
+      alert(`"beeApiUrl" does not start with either "http://" or "https://". Got: ${globalState.beeApiUrl}'`)
+
+      return
+    }
+
+    changeGlobalState({ type: 'BEE_API_URL_SAVE', newValue: globalState.beeApiUrl })
```

Comment:
I think it would be better to stick with calling this `dispatch`, it's more idiomatic in the React world.